abc.templates/infra/backend.tf

```diff
@@ -0,0 +1,6 @@
+terraform {
+  backend "gcs" {
+    bucket = "REPLACE_BUCKET_NAME"
+    prefix = "REPLACE_BUCKET_PREFIX"
+  }
+}
```

Comment:
Its best to be explicit about the providers you need, see https://github.com/abcxyz/gcp-org-terraform-template/blob/main/templates/deployment/terraform.tf#L23-L31

Same goes for terraform version

Reply:
Noticed, there is a separate file, I would merge the backend.tf file and the terraform.tf file into the terraform.tf file.